tox.ini

```diff
@@ -37,6 +37,8 @@ norecursedirs = .* docs requirements
 deps =
     django111: Django>=1.11,<2.0
     django20: Django>=2.0,<2.1
+    django21: Django>=2.1,<2.2
+    django22: Django>=2.2,<2.3
     -r{toxinidir}/requirements/test.txt
 commands =
     py.test {posargs}
```

Comment:
Thanks for the reminder on pytest catching deprecation warnings during test execution by default now, I'd forgotten about that.  But the last time I checked, it didn't start capturing warnings until executing the tests themselves, so it missed (for example) deprecation warnings triggered while loading the Django settings file as part of the `pytest-django` plugin initialization.  So I think we still want this line prefixed with `python -Wd -m` to make sure any such initialization-time deprecation warnings get displayed before the tests start running.